data/formats-data.js

```diff
@@ -817,7 +817,7 @@ exports.BattleFormatsData = {
 	},
 	gengarmega: {
 		randomBattleMoves: ["shadowball","sludgewave","focusblast","taunt","destinybond","disable","perishsong","protect"],
-		randomDoubleBattleMoves: ["shadowball","sludgebomb","focusblast","thunderbolt","substitute","disable","taunt","hypnosis","gigadrain","trick","dazzlinggleam","protect"],
+		randomDoubleBattleMoves: ["shadowball","sludgebomb","focusblast","substitute","disable","taunt","hypnosis","willowisp","dazzlinggleam","protect"],
```

Comment:
Wait, is it really banned? I can't find anything in `formats.js` that would indicate such a thing.